vcr/credential/formats.go

```diff
@@ -26,6 +26,10 @@ func DIFClaimFormats(formats map[string]map[string][]string) Formats {
 		ParamAliases: map[string]string{
 			// no aliases for this type
 		},
+		FormatAliases: map[string]string{
+			"jwt_vp_json": "jwt_vp",
```

Comment:
would you guys still need this after merge? (New PoCs?) Since we were hoping to postpone this issue to when the draft versions are updated and are aligned with each other (PX and OpenID4VC)

Reply:
This is already merged.